fabric-resource-loader-v0/src/client/java/net/fabricmc/fabric/mixin/resource/loader/client/GameOptionsMixin.java

```diff
@@ -132,4 +135,10 @@ private void onLoad(CallbackInfo ci) {
 
 		this.resourcePacks = new ArrayList<>(resourcePacks);
 	}
+
+	@ModifyExpressionValue(method = "refreshResourcePacks", at = @At(value = "INVOKE", target = "Lnet/minecraft/resource/ResourcePackProfile;isPinned()Z"))
```

Comment:
Should this be a WrapOperation? (see https://github.com/FabricMC/fabric/pull/4693#discussion_r2146943357)